README

```diff
@@ -302,6 +327,15 @@ threads to CPUs associated with NUMA node 0.  With this configuration
 standard allocations will be fetched across the QPI bus, and high
 bandwidth allocations will be local to the process CPU.
 
+## Identification of High Bandwidth Memory
+High bandwidth memory are identified using hwloc if available. The default
```

Comment:
Please correct to (mind the comma too):
High bandwidth memory is identified using hwaloc, if available

Reply:
Done in #507 ( after merge #507 it will be available in this PR)